Bonsai/Bonsai.csproj

```diff
@@ -53,7 +58,7 @@
     <EmbeddedResource Include="$(PkgSystem_Runtime_CompilerServices_Unsafe)/lib/net461/System.Runtime.CompilerServices.Unsafe.dll" Visible="false" />
   </ItemGroup>
 
-  <Target Name="NuGetConfig" AfterTargets="Build">
+  <Target Name="NuGetConfig" AfterTargets="PostBuildEvent">
```

Comment:
I should have documented this, thanks. I will try again but at some point I was getting a `NuGet.config` popping up into the project root folder and showing up as an untracked file on git. I believe this was what fixed it, but I will try again. If it replicates I will add a comment, otherwise will revert.

Reply:
If it happens again, try replacing the `TargetDir` with `OutputPath`. (This also shouldn't matter but afaik `TargetDir` is an anachronism and shouldn't really be used in SDK-style projects. Maybe just swap it out anyway since it's kinda weird we use `TargetDir` here.)

Reply:
This seems to have worked, I respect how you seem to have a feeling that "it's kinda weird". It still makes little sense to me, but it is nice that we can keep this step in the `Build` target.